### ISSUE TITLE
Frontend can start on a port different to the one in the backend's .env configuration

### ISSUE THREAD

**Issue by Laurin-W** [assemblee-virtuelle/activitypods#249]
In https://github.com/assemblee-virtuelle/activitypods/issues/245#issue-2338645962 it was reported that when you start the APosd frontend on port 5000 and it is in use already, the cli allows you to use a different port (5001). This works fine but the backend will still be configured to redirect to port 5000 which subsequently causes trouble.
We should make the frontend not start and issue a note that you might want to reconfigure the .env file in case you change the port.
Or we come up with a different solution.

**Comment by srosset81** [comment]
Yes we should probably disable the suggestion to use another port.

**Comment by mgood** [comment]
Yeah, I was trying to test out the project following the tutorial to start it locally in Docker. However, it seems that Mac now uses port 5000 for AirPlay, so it's not as simple as stopping another app since this port is now used by the operating system:
https://chrishannah.me/monterey-blocking-ports-5000-and-7000/

I attempted to change the port forwarding in `docker-compose-dev.yml` to `5001:5000` which does give me access to the frontend, but it just says:
> The Pod provider is currently offline. We apologize for the inconvenience.

It does seem like there's a separate issue where the backend is not accessible on port 3000. However, it does seem like now that Mac is using port 5000, it would be helpful to choose a different default port for the frontend to help simplify the process of getting started.

**Comment by Laurin-W** [comment]
@mgood Maybe your docker engine does not enable network-mode `host` by default. Can you access port 3000 from your host machine? For windows, network-mode `host` is disabled by default and that could be the reason why the frontend cannot access the backend.

Let me know, if this fixes your problem :)

---

On the other side, we should probably change the ports that we use for frontend and backend to more "random" ports.

---

Will investiagte later.
